wjt/xdg-desktop-portal
# print typo

- Clarify the docs on filters in the file chooser
- document-portal: Minor optization for refcounts
- document-portal: Add some internal docs
- document-portal: Improve debugging of LOOKUP/FORGET
- document-portal: Fuse, try to keep down nr of open fds
- document-portal: Don't invalidate forgotten inodes
- document-portal: Fix volatile app ids in fuse test
- document-portal: Ensure root docdir inode alive while there are other inodes
- document-portal: Make Temp files keep the inode around
- document-portal: Try to make inode nrs persistent
- document-fuse: Don't emit invalidation if we unmounted
- document-portal: Fix key error in the tests
- document-portal: Fix compilation with musl
- tests: Disable fuse tests if fuse is not available
- 1.7.2
- Add ru.po translation
- open-uri: Return errors from launch_application_with_uri()
- open-uri: Reject more mismatched open requests
- open-uri: Print debug information when OpenFile/URI fails
- tests: Enable i18n support
- tests: Don't rely on PATH to launch uninstalled
- tests: Print the full paths of what we're trying to run
- tests: Ensure XDG_DATA_DIRS is well-formed
- tests: Make sure http2 test always runs
- tests: Increase timeouts when running in CI
- open-uri: Allow skipping chooser for more URL types
- open-uri: Always open URIs in default app
- open-uri: Show app chooser when default app does not exist
- open-uri: avoid criticals when app to open does not exist
- doc: Include secret portal documentation
- FileChooser: Add a 'current_filter' return value
- Screencast: document source_type property in the response
- Fix typos in FileChooser doc
- Add Russian translation to LINGUAS
- Revert "Add the directory option to the file chooser impl interface"
- camera: make client-node as visible
- screen-cast: fix array element size for permissions
- pipewire: clean up the registry proxy
- request: Close file descriptors attached to the request
- Fail early when a request contains both fd and uri
- Unset fd from requests when it's been consumed
- open-uri: Don't crash if there is no default or latest app
- open-uri: Assert that find_recommended_choices yields non-NULL
- tests: Make app chooser backend more debuggable
- open-uri: Use correct app ID to determine whether it exists
- xdp: Send messages to stderr, not stdout
- tests: Divert service stdout to stderr
- tests: use kill from PATH
- Install session.conf.in to expected path for installed-tests
- tests: call test-document-fuse.py directly
- tests: Only run xdg-document-portal in foreground when uninstalled
- test-document-fuse: Quote more defensively
- tests: Send SIGTERM to dbus-daemon
- CI: Correct the name of the GLib package
- CI: Run installed-tests
- Fix a compiler warning
- Fix distcheck
- Update pofiles for release
- 1.8.0
- Update Swedish translation
- po: update slovak translation
- Remove unused gunixfdlist.h includes
- xdp-utils: Add a g_auto() type to close file descriptors
- open-uri: Always set "fd" before calling handle_open_in_thread_func
- open-uri: Close fd after opening dir or file
- Update Danish translation
- Update Simplified Chinese translation
- build: drop unneeded fontconfig dependency
- Use cgroups to identify snap confined applications rather than AppArmor
- Fix xdp_app_info_has_network implementation for snaps
- Make xdp_app_info_load_app_info() load correct desktop file for snaps
- tests: add some simple tests for the /proc/$pid/cgroup parsing code
- xdp-utils: check if alternate doc portal path matches in path_for_fd()
- document-portal: add snap support to app_has_file_access()
- documents: Add get_real_path_for_doc_path()
- open-uri: Use real filesystem path for OpenDirectory
- wallpaper: Stop passing NULL as the options in the request's response
- tests: Factor out check_fuse() into a file that can be shared by Flatpak
- tests: Skip FUSE tests if /etc/mtab doesn't exist
- 1.8.1
- Settings: do not put failed implementations into the list
- Settings: add basic test to verify backend existence
- Add a security policy
- permission-store: Don't crash when deleting permissions
- utils: Remap /run/flatpak/app, for Flatpak 1.11.x
- ScreenCast: Add 'VIRTUAL' source type
- Quell a compiler warning
- Minor docs clarification
- permission-store: Add GetPermission
- verify_proc_self_fd: Raise a GError
- xdp_app_info_get_path_for_fd: Factor out check_same_file
- utils: Add fallback definition for G_DBUS_METHOD_INVOCATION_HANDLED
- email: Handle error from xdp_app_info_get_path_for_fd
- open-uri: Emit debug messages even if request was not exported yet
- trash: Log debug messages on failure
- xdp_app_info_get_path_for_fd: Raise a GError
- Update LINGUAS
- Added Dutch translation
- location: Update accuracy translation
- ScreenCast: Clarify that some options are optional
- ScreenCast: Fix VIRTUAL source type typo
- utils: Remap Flatpak /var/config, /var/data
- data: add PowerProfileMonitor interface definition
- power-profile-monitor: Add portal implementation for GPowerProfileMonitor
- ScreenCast: Update documentation to Pipewire 0.3 API
- Camera: Update documentation to Pipewire 0.3 API
- Fix typo in Print portal documentation
